src/test/java/tech/aroma/thrift/services/ClientsIT.java

```diff
@@ -67,7 +67,7 @@ public void testNewNotificationServiceClient() throws Exception
     @Test
     public void testNewAromaServiceClient() throws Exception
     {
-        AromaService.Client client = Clients.newAromaServiceClient();
+        AromaService.Client client = Clients.INSTANCE.newAromaServiceClient();
```

Comment:
Make sure that this instance object is not needed, and that it is a static method.